build.gradle

```diff
@@ -37,7 +37,7 @@ ext {
     // Migrate to new startForeground method and test the DataCapturing extensively before
     // targeting version 26! See details here:
     // https://stackoverflow.com/questions/43251528/android-o-old-start-foreground-service-still-working
-    targetSdkVersion = 25
```

Comment:
I guess you did follow the new guidelines on how to start a foreground service in API 26+? In this case you can drop the comment above this line.

Reply:
Jepp. Seems to work good.